### DISCUSSION TITLE
4.4.0 Mac OS M1 install Issue

### DISCUSSION THREAD

**Discussion by sparkypoo** [nextgenhealthcare/connect#5890] [General]
I'm using Mac OS Ventura on an M1 MacMini. When I run the mirthconnect-4.4.0.b2948-macos.dmg installer (or any installer version for that matter) I get an error stating that I need at least 1.8 Java Virtual Machine. I am running:

java version "1.8.0_381"
Java(TM) SE Runtime Environment (build 1.8.0_381-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.381-b09, mixed mode)
---
Matching Java Virtual Machines (2):
    1.8.381.09 (arm64) "Oracle Corporation" - "Java" /Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home
    1.8.0_381 (arm64) "Oracle Corporation" - "Java SE 8" /Library/Java/JavaVirtualMachines/jdk-1.8.jdk/Contents/Home

How can I get around this and run the installer for connect?

**Comment by jonbartels** [comment]
Workarounds:
1. Run the Docker container from https://github.com/nextgenhealthcare/connect-docker
2. Don't use the installer. Just get the `tar.gz` and unpack it somewhere
